server/service/src/log_service.rs

```diff
@@ -0,0 +1,85 @@
+use anyhow::Error;
+use repository::{KeyValueStoreRepository, KeyValueType, RepositoryError};
+use std::{fs, path::Path};
+
+use crate::{service_provider::ServiceContext, settings::Level};
+
+pub trait LogServiceTrait: Send + Sync {
+    fn get_log_file_names(&self) -> Result<Vec<String>, Error> {
+        let log_dir = Path::new("log");
+        let mut log_file_names = Vec::new();
+
+        for entry in fs::read_dir(log_dir)? {
+            let path = entry?.path();
+            log_file_names.push(path.file_name().unwrap().to_string_lossy().to_string());
+        }
+
+        Ok(log_file_names)
+    }
+
+    fn get_log_content(&self, file_name: Option<String>) -> Result<(String, Vec<String>), Error> {
+        let log_dir = Path::new("log");
+        let default_log_file = "remote_server.log".to_string();
```

Comment:
This will always be the latest log file

Reply:
log file name is also configured in yaml files